satpy/tests/reader_tests/test_fci_l1c_nc.py

```diff
@@ -57,9 +57,135 @@
         "scale_factor": 5.58871526031607e-05,
         "add_offset": 1.55617776423501e-01,
     },
+    "3km": {
+        "nrows": 67,
+        "ncols": 3712,
+        "scale_factor": 8.38307287956433e-05,
+        "add_offset": 0.155631748009112,
+    },
 }
 
+LIST_CHANNEL_SOLAR = ["vis_04", "vis_05", "vis_06", "vis_08", "vis_09",
+                          "nir_13", "nir_16", "nir_22"]
+LIST_CHANNEL_TERRAN = ["ir_38", "wv_63", "wv_73", "ir_87", "ir_97", "ir_105",
+                           "ir_123", "ir_133"]
+LIST_TOTAL_CHANNEL = LIST_CHANNEL_SOLAR + LIST_CHANNEL_TERRAN
+LIST_RESOLUTION_V06 = ["1km","3km"]
+LIST_RESOLUTION = ["3km"]
+EXPECTED_POS_INFO_FOR_FILETYPE = {
+        "fdhsi": {"1km": {"start_position_row": 1,
+                          "end_position_row": 200,
+                          "segment_height": 200,
+                          "grid_width": 11136},
+                  "2km": {"start_position_row": 1,
+                          "end_position_row": 100,
+                          "segment_height": 100,
+                          "grid_width": 5568}},
+        "hrfi": {"500m": {"start_position_row": 1,
+                          "end_position_row": 400,
+                          "segment_height": 400,
+                          "grid_width": 22272},
+                 "1km": {"start_position_row": 1,
+                         "end_position_row": 200,
+                         "grid_width": 11136,
+                         "segment_height": 200}},
+        "fci_af" : {"3km": {"start_position_row": 1,
+                          "end_position_row": 67,
+                          "segment_height": 67,
+                          "grid_width": 3712
+                          },
+                    },
+        "fci_af_vis_06" : {"3km": {"start_position_row": 1,
+                          "end_position_row": 67,
+                          "segment_height": 67,
+                          "grid_width": 3712
+                          },
+                        "1km": {"start_position_row": 1,
+                         "end_position_row": 200,
+                         "grid_width": 11136,
+                         "segment_height": 200}
+                    }
+    }
+
+_CHANS_FDHSI = {"solar": LIST_CHANNEL_SOLAR,
```

Comment:
is there a reason for the leading underscore? does this variable need to be "private"

Reply:
The reason is simple I have just used the same convention that was used for these variables it they had underscores. I can erase it if you want.

Reply:
ok, maybe @ameraner can comment, but i don't see the purpose of these variables having a leading underscore.

Reply:
indeed, we can uniformise with the variables above